tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -122,6 +122,8 @@ def signout(self):
         status = response.json().get('data', {}).get('signout', {}).get('status')
         if status == 'SUCCESS':
             print(f"Signed out of {self.url}")
+            self.username = 'anonymous'
```

Comment:
I agree. I'm so sure about the use of `anonymous` as the username after sign out. This implies that a you can access the schema without logging in (i.e., the schema is public). Perhaps it would be better to keep the username as is (as defined by the `signin`) and update the message: `User <username> is signed out` or something like that.